core/src/apps/common/cbor.py

```diff
@@ -40,7 +40,7 @@
 _CBOR_RAW_TAG = const(0x18)
 
 
-def _header(typ: int, l: int) -> bytes:
+def header(typ: int, l: int) -> bytes:
```

Comment:
Cardano's LazyCollection is IMHO too "ubiquitous" to expose these as non-underscored method, making this shared cbor library less consistent than it can othwewise be. Instead I'd just duplicate these within the file declaring the LazyCollection, to contain its implementation details